modules/yieldoneBidAdapter.js

```diff
@@ -21,15 +21,23 @@ export const spec = {
       const cb = Math.floor(Math.random() * 99999999999);
       const referrer = encodeURIComponent(utils.getTopWindowUrl());
       const bidId = bidRequest.bidId;
+      const unitCode = bidRequest.adUnitCode;
       const payload = {
         v: 'hb1',
         p: placementId,
         cb: cb,
         r: referrer,
         uid: bidId,
+        uc: unitCode,
         t: 'i'
       };
 
+      if (window.PREBID_TIMEOUT) {
+        payload.tmax = window.PREBID_TIMEOUT;
+      } else if (window.pbjsTimeout) {
+        payload.tmax = window.pbjsTimeout;
+      }
```

Comment:
Hello @kusapan 

One suggestion is you can utilize the PBJS `config` in order to also get the bidderTimeout.

This value will always be set.

`payload.tmax = config.getConfig('bidderTimeout');`

example: https://github.com/prebid/Prebid.js/blob/8635c91942de9df4ec236672c39b19448545a812/modules/oneplanetonlyBidAdapter.js#L33

This value is set by the publisher by:

pbjs.setConfig({
   bidderTimeout: 850
});

If it is not explicitly set by the publisher, then it defaults to `3000`.

This number is the milliseconds that Prebid gives it's bidders to come back in time for the auction.

So I think it is a good value to send as your `tmax`.

If there are other reasonings or use cases I am unaware of as to why it would not be included please let me know!

Reply:
Our team talks about how to handle tmax.
In this PR, tmax was deleted.